launch/launch/frontend/expose.py

```diff
@@ -98,7 +97,15 @@ def expose_impl_decorator(exposed):
                     name
                 )
             )
-        parse_methods_map[name] = found_parse_method
+        if exposed_type == 'action':
+            # For actions, validate that the user didn't provide unknown attributes or children
+            def wrapper(entity, parser):
```

Comment:
@ivanpauno consider:
```suggestion
            @functools.wraps(found_parse_method)
            def wrapper(entity, parser):
```

Reply:
Done in https://github.com/ros2/launch/pull/468/commits/7367cf9119dc6aa6f6edf1f556a8e1ad2d44233f